CONTRIBUTING.md

```diff
@@ -1,10 +1,40 @@
-# Contributing
+# Create a new Issue
+- Click on the "New issue" button </br>
+ <img src="./public/assets/create_issue.png" /> 
 
-When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.</br>
-<br>Please note we have a code of conduct, please follow it in all your interactions with the project.
+- Give a suitable title regarding the issue like "Update Readme.md", "Replace React context API with Redux", etc.
 
-## Pull Request Process
-- Ensure any install or build dependencies are removed before the end of the layer when doing a build.
-- Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
-- Increase the version numbers in any examples files and the README.md to the new version that this pull request would represent.
-- You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+- Describe your issue. If you think there is need of any image for better explanation, attach image too.
+
+- Then briefly explain what are going to do to solve the issue.
+# How to start with open source and raise your 1st PR
+
+- fork the repository <br />
+  <br /><img src = "./public/assets/fork.png" /> <br />
+
+- clone the repository using Git command <br />
+  <br/> <img src="./public/assets/clone.png" /> <br/>
+  &nbsp; git clone https://github.com/{github_username}/Next-blog-site.git
+- move to the newly created folder using
+  Git command <br />
+  &nbsp; ```cd Next-blog-site/```
+- open in your text editor maually or write command <br> ```code .```
+- make a new branch (always name your branch related to your issue) and then make the changes.<br/> For making new branch , use this Git command
+   <br/> &nbsp; ```git checkout -b {new branch name}```
+- check your changes using command
+<br/> &nbsp; ```git status```
+- save your changes by using command <br/>
+&nbsp; ``` git add .```
+- commit your changes using command. Make sure to write short explaination of your changes in commit message.<br/>
+ &nbsp; ```git commit -m "your_commit_message"```
+- to push your changes to github. Use command
+<br/> &nbsp; ```git push -u origin {branch name}``` 
+
+## Raising Your 1st PR <br/>
+
+- After pushing your code, go to the forked repository. There you will see a section like this. <br />
+ <br><img src="./public/assets/pullreq.png"/>
+
+- click on compare and pull request. Such page will appear. Write meaningfull message and title which describes your changes (title should contain the issue number which you can see beside the title of issue)<br/>
+<br/> <img src="./public/assets/PR.png"/>
+- click on create pull request. <br/>
```

Comment:
All sentences should start with a capital letter.

Reply:
As you can see here all the points start with small letters just change them to capital letter. @Abhirv08

Reply:
sorry, I will fix it soon.😥

Reply:
No worries